test/build.info

```diff
@@ -858,6 +859,11 @@ IF[{- !$disabled{tests} -}]
   INCLUDE[params_test]=.. ../include ../apps/include
   DEPEND[params_test]=../libcrypto.a libtestutil.a
 
+  PROGRAMS{noinst}=param_trie_test
+  SOURCE[param_trie_test]=param_trie_test.c
```

Comment:
Add `../crypto/param_trie.c`

Reply:
No wait, that shouldn't be necessary...

Reply:
I linked against libcrypto.a :)

Reply:
The question about this being public will likely change this.